ladybug_comfort/hi.py

```diff
@@ -94,3 +94,44 @@ def heat_index_warning_category(hi):
         category = 4
 
     return category
+
+
+def humidex(ta, tdp):
+    """Calculate Humidex  from air temperature and the Dew Point.
+
+    The Humidex is a Canadian innovation that was first used in 1965.
+    It describes how hot, humid weather feels to the average person.
+    The Humidex combines the temperature and humidity into one number to reflect the perceived temperature.
+    Because it takes into account the two most important factors that affect summer comfort, it can be a better
+    measure of how stifling the air feels than either temperature or humidity alone. [1]
+
+    The air temperature should be above 20c or the given result is implicitly comfortable.
+    Humidex values range from 20 and up with the following degree of comfort guidelines:
+        20 to 29: Little discomfort
+        30 to 39: Some discomfort
+        40 to 45: Great discomfort; avoid exertion
+        Above 45: Dangerous; heat stroke possible
```

Comment:
I saw that you already mentioned this in one of your comments but it would be useful to have a separate method that returns the category of discomfort similar to the `heat_index_warning_category` function that you can see [here in this module already](https://github.com/ladybug-tools/ladybug-comfort/blob/master/ladybug_comfort/hi.py#L59-L96). This information you wrote here can go into the docstring of that method.

Reply:
Added with https://github.com/ladybug-tools/ladybug-comfort/pull/41/commits/0099bfd8dba3e321cd707c6df52a5021ed19d2a5